frontend_lib/src/index.dev.js

```diff
@@ -3,39 +3,17 @@ import ReactDOM from 'react-dom'
 import './i18n.js'
 import 'core-js/stable'
 import 'regenerator-runtime/runtime'
-import { appContentFactory } from '../src/appContentFactory.js'
-
-const testComponent = props => {
-  props.appContentSaveNewComment(
-    { content_id: 1, workspace_id: 1 },
-    false,
-    'new comment that includes a mention here @admin which should be wrapped',
-    [],
-    () => { console.log('setState called') },
-    'html-document',
-    'foo'
-  )
-
-  props.appContentSaveNewComment(
-    { content_id: 2, workspace_id: 1 },
-    true,
-    'Another comment that contains HTML and a mention @admin which should be wrapped too',
-    [],
-    () => { console.log('setState called') },
-    'html-document',
-    'foo'
-  )
-  return (
-    <div>
-      I'm a dummy test component
-    </div>
-  )
-}
-const WrapperTestComponent = appContentFactory(testComponent)
+import { Avatar, AVATAR_SIZE } from './component/Avatar/Avatar.jsx'
```

Comment:
Maybe add a NOTE to explain that any component can be tested by editing this file?

Reply:
indeed
I'm still not convinced about how to handle this file
I think it should be a .sample to allow dev to update it without pushing it

Reply:
I agree with you.

Reply:
fixed
I added a .sample file and some doc in it